### PR TITLE
Translate strings of federation admin section

### LINKED ISSUE #22767: Missing translations in settings-admin.php
### Steps to reproduce
1. Select a non-English language
2. Activate the Federation app
3. Go to the Admin page, section Federation
4. Observe the missing translations

### Expected behaviour
All text should be translated.

### Actual behaviour
The following are not translated:
- "Add server automatically once a federated share was created successfully" checkbox text
- "Trusted ownCloud Servers" heading
- "+ Add ownCloud server" button text
- "ownCloud server" in edit field after clicking the "+ Add ownCloud server" button  

### Server configuration
**Operating system**: 
Linux

**Web server:**
Nginx

**Database:**
SQLite

**PHP version:**
5.6

**ownCloud version:** (see ownCloud admin page)
ownCloud 9.0.0 beta 2 (daily) Build:2016-03-01T04:41:06+00:00 52d217d77519ed95a18237b09a351f83a0ae7f47

**Updated from an older ownCloud or fresh install:**
Fresh

**Where did you install ownCloud from:**
Daily archive download.

**Signing status (ownCloud 9.0 and above):**

```
N/A
```

**List of activated apps:**

```
Federation, standard
```

**The content of config/config.php:**

```
N/A. Rest of template truncated, all N/A
```

EDIT: The strings are not in Transifex because they are not in t() calls in settings-admin.php.

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #22767 

@karlitschek for 9.0.1 approval

cc @schiesbn @PVince81

### CODE REVIEW

**Comment by karlitschek** [comment]
please backport

File: apps/federation/templates/settings-admin.php [hunk 1]
```diff
@@ -12,13 +12,13 @@
...
-		<label for="autoAddServers">Add server automatically once a federated share was created successfully</label>
+		<label for="autoAddServers"><?php p($l->t('Add server automatically once a federated share was created successfully')); ?></label>
 	</p>
 
-	<h3>Trusted ownCloud Servers</h3>
+	<h3><?php p($l->t('Trusted ownCloud Servers')); ?></h3>
 	<p id="ocFederationAddServer">
-		<button id="ocFederationAddServerButton" class="">+ Add ownCloud server</button>
-		<input id="serverUrl" class="hidden" type="text" value="" placeholder="ownCloud Server" name="server_url"/>
+		<button id="ocFederationAddServerButton" class=""><?php p($l->t('+ Add ownCloud server')); ?></button>
```
Comment (rullzer) [inline apps/federation/templates/settings-admin.php]: Do we really want to also translate the + sign?

Comment (LukasReschke) [inline apps/federation/templates/settings-admin.php]: It's mostly the order. Some languages might want to have this at the end of sentences.

**Comment by rullzer** [comment]
Anyway... fine with me. :+1:

File: apps/federation/templates/settings-admin.php (hunk 1 above)
Comment (rullzer) [inline apps/federation/templates/settings-admin.php]: Yeah ok true.

**Comment by DeepDiver1975** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
lets merge this to get this translated asap

**Comment by MorrisJobke** [comment]
@nickvergessen Could you create the backport PRs?

**Comment by nickvergessen** [comment]
Of this PR? Does not matter too much. The question is if we want to backport the actual language strings. But I would wait with that until 9.0.1

I will create a new issue, to backport this fix + the translations for 9.0.1

**Comment by nickvergessen** [comment]
Done in https://github.com/owncloud/core/issues/22916

**Pull request merged**